examples/deployment-manager/autogen/multivm/README.md

```diff
@@ -1,84 +1,73 @@
+This example uses the `mpdev` tool to generate a Deployment Manager template for a Redis multiple virtual machine (VM).
```

Comment:
> for a Redis multiple virtual machine (VM).

for a Redis multiple virtual machine (VM) deployment.

Reply:
I think this makes sense without the word "deployment," as the intention is to say it's for a VM product (this is how we worded it in the other readme).

Reply:
But for good balance I added the word "product"